engine/parser.def

```diff
@@ -47,13 +50,16 @@ steptextafterlines
   = steptextafterline* { return { "steptextafterlines": text()}; }
 
 steptextafterline
-  = !"====" string __
+  = !"====" !"[step]" string __
  
 steplines
   = stepline+ { return { "steplines": text()}; }
   
 stepline
-  = !"--" string __
+  = function __
+
+function "function call"
+  = [^\r\n(]+[(][^\r\n)]*[)] { return text(); }
```

Comment:
Shouldnt this be os agnostic line breaks?
I would recommend to add a new token for line breaks and whitespaces by default to cover all cases like this :
https://github.com/maybeec/grammars-v4/blob/master/python3/Python3.g4#L612

Reply:
As the line breaks are in [ ] they are os agnostic because this only means "not \r or \n". So if there is only a \n it will also work.